bitnami/contour/values-production.yaml

```diff
@@ -45,19 +45,19 @@ configInline:
   # disable ingressroute permitInsecure field
   disablePermitInsecure: false
   tls:
-  #   minimum TLS version that Contour will negotiate
-  #   minimum-protocol-version: "1.1"
-  # Defines the Kubernetes name/namespace matching a secret to use
-  # as the fallback certificate when requests which don't match the
-  # SNI defined for a vhost.
+    # minimum TLS version that Contour will negotiate
+    # minimum-protocol-version: "1.1"
+    # Defines the Kubernetes name/namespace matching a secret to use
+    # as the fallback certificate when requests which don't match the
+    # SNI defined for a vhost.
     fallback-certificate:
   #   name: fallback-secret-name
   #   namespace: '{{ .Release.Namespace }}'
```

Comment:
I think this is still wrong

Reply:
If I put it in the other way, the linter is failing.